gulpfile.js

```diff
@@ -121,7 +121,7 @@ var codegen = function(project, cb) {
                         ' --azure-arm ' +
                         generator +
                         ` --namespace=${mappings[project].package} ` +
-                        ` --output-folder=${outDir} ` +
```

Comment:
I had a slightly different fix for this: https://github.com/Azure/azure-libraries-for-java/pull/159/files#diff-b9e12334e9eafd8341a6107dd98510c9L123